### ISSUE TITLE
Panel: when replacing an image, the image is replaced but keeps its name

### ISSUE THREAD

**Issue by bastianallgeier** [k-next/starterkit#34]

**Comment by bastianallgeier** [comment]
@texnixe this is intended behavior. What would you expect to happen?

**Comment by texnixe** [comment]
Oh, I wasn't  aware of the fact that Kirby 2 had the same sort of behavior. Especially in a collection I'd expect I could just swap an image for another (especially with the cover image in a project). But then this would mean that the behavior was different from file view, hm :thinking:

**Comment by bastianallgeier** [comment]
You are right! I think it would be fantastic if the replace option in the dropdown would open a little dialog, asking if you want to replace the file or swap it in the collection.

**Comment by bastianallgeier** [comment]
I'll convert this into an idea.
